recipes/oatpp-websocket/all/test_package/CMakeLists.txt

```diff
@@ -4,9 +4,6 @@ project(test_package)
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup()
 
-find_package(oatpp REQUIRED CONFIG)
-find_package(oatpp-websocket REQUIRED CONFIG)
-
-add_executable(${PROJECT_NAME} test_package.cpp FullAsyncTest.cpp)
-target_link_libraries(${PROJECT_NAME} oatpp::oatpp oatpp::oatpp-test oatpp::oatpp-websocket)
+add_executable(${PROJECT_NAME} test_package.cpp)
```

Comment:
Why did you revert this?
This tests the components in `package_info`.

Reply:
Got it, brought back.